File: qubit/python/complex.py
class Complex:
    """
    This class models a complex number
    """

    def __init__(self, real_number, imaginary_number):
        self.real_number = real_number
        self.imaginary_number = imaginary_number

    def module(self):
        return pow( pow(self.real_number, 2) + pow(self.imaginary_number, 2), 0.5)

    def get_complex_value(self):
        return str(self.real_number) + " + " + str(self.imaginary_number) + "i"

    def set_values(self, real_number, imaginary_number):
        self.real_number = real_number
        self.imaginary_number = imaginary_number

File: qubit/kotlin/Complex.kt
import kotlin.math.pow

/**
 * This class models a complex number
 * @author Paul Rodriguez-Ch
 */
class Complex (realNumber: Float, imaginaryNumber: Float){

    var realNumber: Float
    var imaginaryNumber: Float

    init {
        this.realNumber = realNumber
        this.imaginaryNumber = imaginaryNumber
    }

    fun module(): Float{
        return (this.realNumber.pow(2) + this.imaginaryNumber.pow(2)).pow(0.5f)
    }

    fun getComplexValue(): String{
        return "" + this.realNumber.toString() + " + " + this.imaginaryNumber.toString() + "i"
    }

    fun setValues(realNumber: Float, imaginaryNumber: Float){
        this.realNumber = realNumber
        this.imaginaryNumber = imaginaryNumber
    }
}